### DISCUSSION TITLE
A little high-five for the sounds used in the plugin!

### DISCUSSION THREAD

**Discussion by areasell** [RedShyGuy/Vapecord-ACNL-Plugin#70] [General]
I'm absolutely adoring the wild world sounds you used for this plugin! Makes everything even the more better! Please never replace those sounds.
-A fan of the wild world sounds in the plugin

**Comment by RedShyGuy** [comment]
I don't plan to replace the sounds. But even if, you don't have to worry, the sounds are placed on the SD-Card, which means you could just replace the sounds with the old ones if I ever replace them. But glad you like them!🙂
